docs/pages/ingesting/ingest_scripts/writing_scipts/writing_scripts.rst

```diff
@@ -0,0 +1,74 @@
+Writing Scripts
+==================
+
+When writing ingest scripts, there are two different ways to go about it: using existing ingest functions from `astrodb_utils` or using sqlalchemy commands. 
+
+
+Using Existing Ingest functions
+---------------------------------
+Using existing ingest functions helps streamline the process of writing an ingest script. 
+However, only few ingest functions exist, namely for sources, names, and instruments. 
+Therefore, if your data fits into one of these categories, it is recommended to use the existing functions. 
+
+Below is an example of how to use the `ingest_source` function to ingest source data into the database:
+
+.. code-block:: python
+
+    for source in bones_sheet_table:
+
+            ingest_source(
+                db,
+                source=source["NAME"],
+                reference=reference[1],
+                ra=source["RA"],
+                dec=source["DEC"],
+                ra_col_name="ra",
+                dec_col_name="dec",
+                epoch_col_name="epoch",
+                raise_error=True,
+                search_db=True,
+                comment="Discovery reference from the BONES archive",
+            )  
+
+Note that the basic structure for any ingest is looping through each row of your data table and appropriately ingesting each row into the database with the relevant parameters. 
+Each ingest function will have different required and optional parameters, so be sure to check the API documentation for more details. 
+
+
+Using SQLAlchemy Commands
+---------------------------------
+If there is no existing ingest function for your data type, you can use sqlalchemy commands to directly ingest into the database. 
+
+Below is an example of how to ingest modeled parameters data into the database using sqlalchemy commands:
+
+.. code-block:: python
+
+    for row in L6T6_table:
+        with db.engine.connect() as conn:
+            conn.execute(
+                db.ModeledParameters.insert().values(
+                    {
```

Comment:
make into a dictonary and then pass the dictionary to the insert method.  Similar to https://astrodbkit.readthedocs.io/en/latest/#adding-data